packages/analytics-browser/src/browser-client.ts

```diff
@@ -26,6 +26,7 @@ export class AmplitudeBrowser extends AmplitudeCore<BrowserConfig> {
     this.initializing = true;
 
     // Step 1: Read cookies stored by old SDK
+    apiKey = apiKey ?? '';
```

Comment:
alternatively, you can also modify the param list and provide a default when none is provided

```
async init(apiKey = '', ...
```